src/design-system/layout/Center/Center.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import centerProps from './Center.types';
+
+const Center: React.FC<centerProps> = ({
+  text = false,
+  gutter = true,
```

Comment:
I wonder whether it's beneficial to use conditional props here over explicit ones. Are we losing much by making them required?

More importantly I think a small explanation of what they do would be beneficial to future travellers:

```suggestion
  /** used to center align text to the horizontal centre */
  text = false,
  /** include horizontal padding around the text */
  gutter = true,
```

Reply:
Yeah, that’s a fair point and makes sense. I’ll update the component to use required props and include some documentation to make the behaviour clearer.